tmt/steps/provision/testcloud.py

```diff
@@ -1384,3 +1399,66 @@ def clean_images(cls, clean: 'tmt.base.Clean', dry: bool, workdir_root: Path) ->
                     clean.fail(f"Failed to remove '{image}'.", shift=2)
                     successful = False
         return successful
+
+
+@container
+class ConsoleLog(tmt.steps.provision.GuestLog):
+    # Linked guest
+    guest: GuestTestcloud
+
+    # Path where testcloud will create the symlink to the console log
+    symlink: Path
+
+    # Temporary directory for storing the console log content
+    tmp: Optional[Path] = None
```

Comment:
I get the idea, I just don't like the naming.

* `tmp` would definitely deserve something better, to express its purpose, as it's not just any temporary directory, it's that "jump host"-like directory, the "exchange" directory, the "middle ground" directory.
* `symlink` - I had to think hard when seeing in code, and I knew what was it for. A file path testcloud would point to the actual log in `tmp`.

Reply:
Renamed to `testcloud_symlink_path` and `exchange_directory`.